src/types/timezone.jl

```diff
@@ -4,6 +4,12 @@
 # to the cache, while still being thread-safe.
 const THREAD_TZ_CACHES = Vector{Dict{String,Tuple{TimeZone,Class}}}()
 
+# Holding a lock during construction of a specific TimeZone prevents multiple Tasks (on the
+# same or different threads) from attempting to construct the same TimeZone object, and
+# allows them all to share the result.
+const tz_cache_mutex = ReentrantLock()
```

Comment:
Let's keep our constants consistent
```suggestion
const TZ_CACHE_MUTEX = ReentrantLock()
```

Reply:
huh, i guess i forgot to push after my last commit. Updated now, finally. Resolving